src/main/resources/assets/catlas/lang/zh_CN.lang

```diff
@@ -7,6 +7,7 @@ catlas.config.map.toggle.show_player_names=显示玩家名称
 catlas.config.map.toggle.vanilla_head_marker=原版头颅标记
 catlas.config.map.size.map_text_scale=地图文本缩放
 catlas.config.map.size.player_heads_scale=玩家头颅缩放
+catlas.config.map.size.head_border_scale=Head Border Scale
```

Comment:
If it's missing it should default to en_US already, so it doesn't need to be added here.